package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hello-microverse",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/richy4dev/Hello-Microverse.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/richy4dev/Hello-Microverse/issues"
+  },
+  "homepage": "https://github.com/richy4dev/Hello-Microverse#readme",
+  "devDependencies": {
+    "hint": "^7.1.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
![MIT-file](https://user-images.githubusercontent.com/93039359/184939413-761e306c-1f97-471e-8f6a-b37144f5c91f.png)

## [Optional]
- [ ] Your are following almost all requirements but you have added a link to MIT.md file but please add an MIT.md file to your project from this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush: because this is your license file so please take it seriously :+1:
- [ ] After adding it don't forget to add your name, current year and app-type in MIT.md file because most of the students forget :thinking: